microbench/db_basic_bench.cc

```diff
@@ -596,36 +597,27 @@ static void DBGet(benchmark::State& state) {
       static_cast<double>(not_found * 100), benchmark::Counter::kAvgIterations);
 
   if (state.thread_index() == 0) {
-    if (enable_statistics) {
-      HistogramData histogram_data;
-      options.statistics->histogramData(DB_GET, &histogram_data);
-      state.counters["get_mean"] = histogram_data.average * std::milli::den;
-      state.counters["get_p95"] = histogram_data.percentile95 * std::milli::den;
-      state.counters["get_p99"] = histogram_data.percentile99 * std::milli::den;
-    }
+    HistogramData histogram_data;
+    options.statistics->histogramData(DB_GET, &histogram_data);
+    state.counters["get_mean"] = histogram_data.average * std::milli::den;
+    state.counters["get_p95"] = histogram_data.percentile95 * std::milli::den;
+    state.counters["get_p99"] = histogram_data.percentile99 * std::milli::den;
 
     TeardownDB(state, db, options, kg);
   }
 }
 
 static void DBGetArguments(benchmark::internal::Benchmark* b) {
-  for (int comp_style : {kCompactionStyleLevel, kCompactionStyleUniversal,
-                         kCompactionStyleFIFO}) {
-    for (int64_t max_data : {128l << 20, 512l << 20}) {
-      for (int64_t per_key_size : {256, 1024}) {
-        for (bool enable_statistics : {false, true}) {
-          for (bool negative_query : {false, true}) {
-            for (bool enable_filter : {false, true}) {
-              b->Args({comp_style, max_data, per_key_size, enable_statistics,
-                       negative_query, enable_filter});
-            }
-          }
+  for (int64_t max_data : {128l << 20, 512l << 20}) {
```

Comment:
It's a little surprising to see that compaction style has little effect on read performance.

Reply:
The 128MB didn't do compaction. The 512MB setting did a few compactions. Compaction style might make a difference in the latter case. But I thought it'd only catch read performance changes at an arbitrary early point in populating an LSM, which didn't feel necessary to measure. I can add it back though if anyone wants.

Reply:
Added back the removed parameters.